src/SafeTensors.jl

```diff
@@ -349,13 +350,39 @@ Eagerly load the tensors in `filename`.
 """
 function load_safetensors(filename::AbstractString; mmap = true)
     safetensor = deserialize(filename; mmap)
-    tensors = Dict{String, Array}(); sizehint!(tensors, length(safetensor))
+    tensors = Dict{String,Array}()
+    sizehint!(tensors, length(safetensor))
     for (name, tensor) in safetensor
         tensors[name] = collect(tensor)
     end
     return tensors
 end
 
-export load_safetensors
+function load_sharded_safetensors(dir::AbstractString; mmap=true)
+    weight_file = joinpath(dir, "model.safetensors")
+    index_file = joinpath(dir, "model.safetensors.index.json")
+
+    if isfile(weight_file)
+        return load_safetensors(weight_file; mmap=mmap)
```

Comment:
1. file names are constants, we need either specify that in the doc or make them variables.
2. If the "model.safetensors" is present, it would ignore "model.safetensors.index.json".
3. missing docstring